db/schema.rb

```diff
@@ -12,13 +12,16 @@
 
 ActiveRecord::Schema.define(version: 2018_09_14_014133) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
I don't think this is necessary for your PR and it conflicts with #288.

Reply:
...on the other hand, this is the schema you actually want to be deployed, right?  or does that not matter, because the schema is never used directly for deployed services, just the chain of migrations?

but i did have the same thought.  and i do think it'd be good to do #288 (makes patch party style work easier, would've saved jeremy the setup headache he ran into yesterday).

Reply:
Yeah, the schema is only used when starting from scratch.  Otherwise, migrations.

Reply:
My inclination is to do whatever is the least work.  #288 is a good idea, but is it critical path?

I included these changes because schema.rb changed just from my running stuff locally.

Whatever y'all decide is fine.  I can take this commit out of this PR just to move this ahead, or whatever.

Reply:
noted this on #288.  we can regenerate schema.rb should we get to that.  that should be a pretty painless additional step.  would rather get this merged now, esp since the current devs on this project do have PG up and running locally at the moment.